models/curvefi/celo/curvefi_celo_trades.sql

```diff
@@ -0,0 +1,105 @@
+{{
+    config(
+        tags = ['dunesql'],
+        alias = alias('trades'),
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
+        post_hook = '{{ expose_spells(\'["celo"]\',
```

Comment:
same question here on exposing the spell (we're leaning towards not exposing many of these upstream spells, but rather just the end result of `dex.trades`)

Reply:
agreed, will remove